metadata-db-app/src/cmr/metadata_db/data/oracle/concepts.clj

```diff
@@ -644,6 +644,31 @@
                 []
                 concept-id-rev-ids-map))))
 
+(defn get-collection-concept-ids
```

Comment:
Missing docstring

Reply:
It's the CMR coding standard that we don't duplicate docstring for implementation functions for a protocol. The docstring is in the protocol function itself which is in `cmr.metadata-db.data.concepts` in this case.